src/main/java/uk/co/ramp/covid/simulation/population/Population.java

```diff
@@ -115,10 +114,10 @@ private boolean householdAllocationImpossible(BitSet adultIndex, BitSet pensione
                 case ADULTPENSIONERCHILD: adultPensionerChild++; break;
             }
         }
-        boolean impossible = adultIndex.cardinality() <= adult + adultPensioner + adultChild + adultPensionerChild
-                || pensionerIndex.cardinality() <= adultPensioner + pensioner + pensionerChild + adultPensionerChild
-                || childIndex.cardinality() + infantIndex.cardinality() <= adultChild + pensionerChild + adultPensionerChild;
-        return  !impossible;
+        boolean possible = adultIndex.cardinality() > adult + adultPensioner + adultChild + adultPensionerChild
+                || pensionerIndex.cardinality() > adultPensioner + pensioner + pensionerChild + adultPensionerChild
```

Comment:
These should be && I think since we need to ensure we have enough of all types of people before we exit the function.